microservice-app/src/main/java/com/sdevelopment/skt/microservice/exception/DuplicatedProductEception.java

```diff
@@ -0,0 +1,11 @@
+package com.sdevelopment.skt.microservice.exception;
+
+public class DuplicatedProductEception extends Exception {
```

Comment:
why not extending from `RunTimeException` ?

Reply:
I extended form Exception because it was the only way I knew until just now but looking into it I see that some people think that catching runtime exceptions is a terrible idea because you'll catch unrecoverable exceptions such as NullPointerException. 
Here I want to deal with an exception that I can't avoid.

Reply:
RunTimeExceptions are not a bad idea, actually, it's correct to use them. I consider you should try to look for information in more reliable sources.
Please read Item 70 of Effective Java, here you'll find more information about Exception handling and when you need to use each type of exception.

Reply:
and also, please check the difference between a Checked vs an unchecked exception